Steer/DigitizerWorkflow/src/ITSMFTDigitWriterSpec.cxx

```diff
@@ -77,16 +77,18 @@ class ITSMFTDPLDigitWriter
     auto inDigits = pc.inputs().get<std::vector<o2::itsmft::Digit>>((detStr + "digits").c_str());
     auto inROFs = pc.inputs().get<std::vector<o2::itsmft::ROFRecord>>((detStr + "digitsROF").c_str());
     auto inMC2ROFs = pc.inputs().get<std::vector<o2::itsmft::MC2ROFRecord>>((detStr + "digitsMC2ROF").c_str());
-    auto inLabels = pc.inputs().get<MCCont*>((detStr + "digitsMCTR").c_str());
+    if (mWithMCTruth) {
+      auto inLabels = pc.inputs().get<MCCont*>((detStr + "digitsMCTR").c_str());
+      auto labelsRaw = inLabels.get();
+      getOrMakeBranch(*mOutTreeDig.get(), (detStr + "DigitMCTruth").c_str(), &labelsRaw);
```

Comment:
This provides a pointer on the local labelsRaw as a branch address, is this safe? I think we to define the ``labelsRaw`` as a pointer in the run() scope.